.rubocop.yml

```diff
@@ -0,0 +1,18 @@
+AllCops:
+  Exclude:
+    - bin/**/*
+    - db/**/*
+    - tmp/**/*
+    - vendor/**/*
+    - log/**/*
+
+  RunRailsCops: true
+  DisplayCopNames: true
+  DisplayStyleGuide: true
+  StyleGuideCopsOnly: true
+
+Style/StringLiterals:
+  EnforcedStyle: double_quotes
```

Comment:
After double checking the style guide, this is the only Cop that I wasn't happy about.

That said I think that you made the right call here. 

Before today we weren't enforcing any style, I'm happy to enforce one, and given that you already modified the code base let's go with it :smile: :bow: